source/common/quic/envoy_quic_client_session.cc

```diff
@@ -34,6 +35,13 @@ EnvoyQuicClientSession::~EnvoyQuicClientSession() {
 
 absl::string_view EnvoyQuicClientSession::requestedServerName() const { return server_id().host(); }
 
+void EnvoyQuicClientSession::dumpState(std::ostream& os, int indent_level) const {
+  const char* spaces = spacesForLevel(indent_level);
+  os << spaces << "Http3::ClientSession" << DUMP_MEMBER(disable_keepalive_);
```

Comment:
Are there any other members we might want to dump, or is disable_keepalive_ the only one?